### PR TITLE
Add autoload for TestLanguage.psm1 TestHelpers.psm1

### PR DESCRIPTION
**Pull request by iSazonov**
The test temporary modules moved to `test\tools\Modules`.
I did two commits to simplify the review:
1. Commit for Test.Helpers.psm1 (was renamed to TestHelpers.psm1)
2. Commit for LanguageTestSupport.psm1 (was renamed to TestLanguage.psm1)

### CODE REVIEW

**Review by lzybkr — COMMENTED** [review]
2 small comments
* you can delete most of the comments from the module manifest, they aren't useful
* can you think of a better name for the module `TestLanguage` - it feels too similar to a hypothetical script or cmdlet. Maybe `LanguageTestHelpers` - and maybe `TestHelpers` should be `CommonTestHelpers`, or something like that.

**Comment by iSazonov** [comment]
Can we add a "Module" suffix?
- TestHelpersModule
- TestLanguageModule
- TestHostCSModule
- TestRemotingModule

**Comment by lzybkr** [comment]
Mostly I'm just thinking out loud here, it just struct me as odd seeing verbs in module names. I was curious how common this was, and it's not uncommon:

```powershell
$verbs = (Get-Verb).Verb; Find-Module | ? { $modName = $_.Name; $verbs | ? { $modName.StartsWith($_) } }
```

This gives my 97 modules, some of which sound more like scripts than modules, but it's hard to see without looking more closely.

At any rate, consider this just one opinion, nothing more.

**Comment by iSazonov** [comment]
I thought about how to easily find these modules using `Get-Module`. A common prefix can help instead of `Test` verb:
- HelpersCommon
- HelpersLanguage
- HelpersHostCS
- HelpersRemoting

**Comment by iSazonov** [comment]
@lzybkr @TravisEz13 In last commit I renamed modules (Removed approved verbs (Get-Verb) from module names). 
Please continue the review.

**Comment by iSazonov** [comment]
@lzybkr @TravisEz13 Do I need something else to do here?

**Comment by iSazonov** [comment]
@lzybkr @TravisEz13 Could you please review and merge? This blocks further work.

File: test/tools/Modules/HelpersCommon/HelpersCommon.psd1 [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+CompanyName = 'Microsoft Corporation'
+
+Copyright = 'Copyright (C) Microsoft Corporation, All rights reserved.'
+
+Description = 'Temporary module contains functions for using in tests'
+
+FunctionsToExport = 'Wait-UntilTrue', 'Test-IsElevated', 'ShouldBeErrorId'
+
+#CmdletsToExport = '*'
```
Comment (TravisEz13) [inline test/tools/Modules/HelpersCommon/HelpersCommon.psd1]: There are still comments here

File: test/tools/Modules/HelpersHostCS/HelpersHostCS.psd1 [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+CompanyName = 'Microsoft Corporation'
+
+Copyright = 'Copyright (C) Microsoft Corporation, All rights reserved.'
+
+Description = 'Simple console host for console IO tests.'
+
+FunctionsToExport = 'New-TestHost'
+
+#CmdletsToExport = '*'
```
Comment (TravisEz13) [inline test/tools/Modules/HelpersHostCS/HelpersHostCS.psd1]: can be removed

File: test/tools/Modules/HelpersLanguage/HelpersLanguage.psd1 [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+CompanyName = 'Microsoft Corporation'
+
+Copyright = 'Copyright (C) Microsoft Corporation, All rights reserved.'
+
+Description = 'Temporary module for language tests'
+
+FunctionsToExport = 'Get-ParseResults', 'Get-RuntimeError', 'ShouldBeParseError',
+                    'Test-ErrorStmt', 'Test-Ast', 'Test-ErrorStmtForSwitchFlag'
+
+#CmdletsToExport = '*'
```
Comment (TravisEz13) [inline test/tools/Modules/HelpersLanguage/HelpersLanguage.psd1]: see previous comment

File: test/tools/Modules/HelpersRemoting/HelpersRemoting.psd1 [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+CompanyName = 'Microsoft Corporation'
+
+Copyright = 'Copyright (C) Microsoft Corporation, All rights reserved.'
+
+Description = 'Temporary module for remoting tests'
+
+FunctionsToExport = 'New-RemoteRunspace', 'New-RemoteSession'
+
+#CmdletsToExport = '*'
```
Comment (TravisEz13) [inline test/tools/Modules/HelpersRemoting/HelpersRemoting.psd1]: see previous comment

**Review by TravisEz13 — COMMENTED** [review]
I don't see any major issues.

File: test/tools/Modules/HelpersHostCS/HelpersHostCS.psd1 (hunk 2 above)
Comment (iSazonov) [inline test/tools/Modules/HelpersHostCS/HelpersHostCS.psd1]: Fixed.

File: test/tools/Modules/HelpersLanguage/HelpersLanguage.psd1 (hunk 3 above)
Comment (iSazonov) [inline test/tools/Modules/HelpersLanguage/HelpersLanguage.psd1]: Fixed.

File: test/tools/Modules/HelpersRemoting/HelpersRemoting.psd1 (hunk 4 above)
Comment (iSazonov) [inline test/tools/Modules/HelpersRemoting/HelpersRemoting.psd1]: Fixed.

**Comment by iSazonov** [comment]
@TravisEz13 Could you please continue with the PR?

**Comment by TravisEz13** [comment]
@iSazonov  Sorry, I've been out of the office for a few weeks.

**Comment by TravisEz13** [comment]
Closed and re-opened PR to re-trigger CI.  Could you rebase as well?

**Comment by iSazonov** [comment]
Rebase done.

**Review by TravisEz13 — APPROVED** [review]

**Comment by iSazonov** [comment]
@TravisEz13 Is the PR ready to merge?

**Comment by TravisEz13** [comment]
Yeah @iSazonov, Thanks.  I was giving people time to give feedback.  Thanks for pinging me too.

**Pull request merged**